db/db.go

```diff
@@ -1107,12 +1143,168 @@ func InsertNewStation(
 		DlsConfigurationPoison:      dlsConfiguration.Poison,
 		DlsConfigurationSchemaverse: dlsConfiguration.Schemaverse,
 		TieredStorageEnabled:        tieredStorageEnabled,
+		TenantName:                  strings.ToLower(tenantName),
 	}
 
 	rowsAffected := rows.CommandTag().RowsAffected()
 	return newStation, rowsAffected, nil
 }
 
+// TODO: check if need to add condition instead duplicate same functions
+func GetAllStationsDetailsPerTenant(tenantName string) ([]models.ExtendedStation, error) {
+	ctx, cancelfunc := context.WithTimeout(context.Background(), DbOperationTimeout*time.Second)
+	defer cancelfunc()
+	conn, err := MetadataDbClient.Client.Acquire(ctx)
+	if err != nil {
+		return []models.ExtendedStation{}, err
+	}
+	defer conn.Release()
+	query := `
```

Comment:
tenant name is not in the query - add it to where section

Reply:
its not should be, the s.* retrieve it